apps/backend/src/config/stfc/configureSTFCEnvironment.ts

```diff
@@ -83,6 +83,7 @@ async function enableDefaultStfcFeatures() {
       FeatureId.STFC_IDLE_TIMER,
       FeatureId.TECHNICAL_REVIEW,
       FeatureId.FAP_REVIEW,
+      FeatureId.USER_SURNAME_SEARCH_FILTER,
```

Comment:
Maybe it is better to name the feature just `USER_SEARCH_FILTER`. Now it is searching by surname but it might change in the future I guess and being able to search by email as well for example.